api/DOMRectList.json

```diff
@@ -0,0 +1,145 @@
+{
+  "api": {
+    "DOMRectList": {
```

Comment:
`DOMRect` and `DOMRectList` are a bit special, they're what `element.getBoundingClientRect()` and `element.getClientRects()` respectively return, and those are very old APIs. But originally they returned objects called `ClientRect` and `ClientRectList`, except those interfaces weren't exposed.

So the support statements should go back at least as far as https://developer.mozilla.org/en-US/docs/Web/API/Element/getClientRects#Browser_compatibility as `alternative_name` statements.

http://software.hixie.ch/utilities/js/live-dom-viewer/saved/8745 should reveal what the name of it in each browser was. Testing it in Firefox 26 I see `ClientRect` and I suspect that's the case for other browsers too.

Can you add these older support statements and check it for consistency with DOMRect.json + DOMRectReadOnly.json as well?

Reply:
Since the API doesn't seem to be exposed at first, I'll recreate this PR with data from a custom test.